### PR TITLE
Add hreflang fields to story page

### PR DESCRIPTION
**Pull request by gustaveen**
Now we get alternates from storyblok instead.

### CODE REVIEW

**Review by palmenhq — DISMISSED** [review]
Awesome!

File: storyblok/components.json [hunk 1]
```diff
@@ -3501,6 +3501,22 @@
...
+          "type": "multilink",
+          "link_scope": "sv/"
+        },
+        "hreflang_no": {
+          "type": "multilink",
+          "link_scope": "no/"
+        },
+        "Hreflang": {
+          "type": "section",
+          "keys": ["hreflang_en", "hreflang_sv", "hreflang_no"]
```
Comment (palmenhq) [inline storyblok/components.json:3519]: Can we kill of this stuff now maybe? Seems it is not used in the `<head>` anymore? (and all the rests from it)

**Review by palmenhq — DISMISSED** [review]
awesome!

File: storyblok/components.json (hunk 1 above)
Comment (gustaveen) [reply storyblok/components.json:3519]: Oops, of course! Thanks for catching that!

**Pull request merged**